src/sentry/conf/server.py

```diff
@@ -1147,14 +1150,19 @@ def get_raven_config():
     'raven-java:log4j2': 'https://docs.sentry.io/clients/java/modules/log4j2/',
     'raven-java:logback': 'https://docs.sentry.io/clients/java/modules/logback/',
     'raven-js': 'https://docs.sentry.io/clients/javascript/',
+    'raven-node': 'https://docs.sentry.io/clients/node/',
     'raven-python': 'https://docs.sentry.io/clients/python/',
+    'raven-ruby': 'https://docs.sentry.io/clients/ruby/',
     'raven-swift': 'https://docs.sentry.io/clients/cocoa/',
     'sentry-php': 'https://docs.sentry.io/clients/php/',
     'sentry-laravel': 'https://docs.sentry.io/clients/php/integrations/laravel/',
+    'sentry-swift': 'https://docs.sentry.io/clients/cocoa/',
 }
 
 DEPRECATED_SDKS = {
     # sdk name => new sdk name
     'raven-objc': 'sentry-swift',
     'raven-php': 'sentry-php',
+    # The Ruby SDK used to go by the name 'sentry-raven'...
```

Comment:
Hmm, but this changed in `raven-ruby:2.4.0`, I feel like this comment is future-proof. What would you want it to say?